### PR TITLE
Fix tooltips on Chromium 114

### PR DESCRIPTION
**Pull request by noahwc**
Chromium 114 introduced a popover utility which conflicted with shoelace's internal popover utility. This PR moved the shoelace utility to it's own namespace.

Process.st:
JIRA: [SQDEV-3436](https://impact.atlassian.net/browse/SQDEV-3436)
Process.st: https://app.process.st/runs/ShoelaceMint-Popover-Fix-jydtlb4Z5gN3QQx0UFVHDQ/tasks/ipJPSwLUytFe3xMt6P5PYw

### CODE REVIEW

**Comment by DerekSiemens** [comment]
I think it would be good for us to start a changelog @noahwc

File: package.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
-  "name": "@shoelace-style/shoelace",
+  "name": "@saasquatch/shoelace",
   "description": "A forward-thinking library of web components.",
-  "version": "2.0.0-beta.32",
-  "homepage": "https://github.com/shoelace-style/shoelace",
+  "version": "1.0.0-0",
```
Comment (00salmon) [inline package.json]:
```suggestion
  "version": "1.0.0",
```

File: package-lock.json [hunk 2]
```diff
@@ -1,8 +1,5366 @@
 {
-  "name": "@shoelace-style/shoelace",
-  "version": "2.0.0-beta.32",
-  "lockfileVersion": 1,
+  "name": "@saasquatch/shoelace",
+  "version": "1.0.0-0",
```
Comment (00salmon) [inline package-lock.json:3]:
```suggestion
  "version": "1.0.0",
```

**Review by 00salmon — COMMENTED** [review]
👍 good to go, just need a version update

**Pull request merged**